modules/apps/depot/depot-test/src/testIntegration/java/com/liferay/depot/service/test/DepotEntryGroupRelLocalServiceTest.java

```diff
@@ -91,13 +101,83 @@ public void testAddDepotEntryGroupRelWithARepeatedDepotEntryGroupRel()
 			finalDepotEntryGroupRel.getToGroupId());
 	}
 
+	@Test
+	public void testAddDepotEntryGroupRelWithSiteTemplateWithoutPropagation()
+		throws Exception {
+
+		DepotEntry depotEntry = _addDepotEntry();
+
+		LayoutSetPrototype layoutSetPrototype =
+			_layoutSetPrototypeLocalService.addLayoutSetPrototype(
+				TestPropsValues.getUserId(), TestPropsValues.getCompanyId(),
+				HashMapBuilder.put(
+					LocaleUtil.getDefault(), RandomTestUtil.randomString()
+				).build(),
+				(Map<Locale, String>)null, true, true, false,
+				ServiceContextTestUtil.getServiceContext());
+
+		Group layoutSetPrototypeGroup = _setUpLayoutSetPrototypeGroup(
+			layoutSetPrototype);
+
+		DepotEntryGroupRel depotEntryGroupRel =
+			_depotEntryGroupRelLocalService.addDepotEntryGroupRel(
+				depotEntry.getDepotEntryId(),
+				layoutSetPrototypeGroup.getGroupId());
+
+		Assert.assertNotNull(depotEntryGroupRel.getDepotEntryId());
```

Comment:
@robertoDiaz, can't we change this assert so as to compare that  deportEntry.getDepotEntryId == depotEntryGroupRel.getDepotEntryId instead of checking if it's not null?

Reply:
I asked because you are assigning the depotEntryId in the service. 

LGTM